src/components/TextField/TextField.tsx

```diff
@@ -8,6 +8,7 @@ type Props = {
   placeholder?: string;
   required?: boolean;
   onChange?: (newValue: string) => void;
+  isUrlValid?: boolean;
```

Comment:
This is good, you try adding this, but in this very case, it makes sense to make `TextField` more generic, and more universal

1/3

```suggestion
  externalError?: string
```

Reply:
Got it, thanks